pkg/repositories/config/migrations.go

```diff
@@ -1,12 +1,22 @@
 package config
 
 import (
+	"database/sql"
+	"fmt"
+
 	"github.com/flyteorg/flyteadmin/pkg/repositories/models"
 	schedulerModels "github.com/flyteorg/flyteadmin/scheduler/repositories/models"
-	gormigrate "github.com/go-gormigrate/gormigrate/v2"
+	"github.com/go-gormigrate/gormigrate/v2"
 	"gorm.io/gorm"
 )
 
+// TODO: add a way to get these list of tables directly from the gorm loaded models
```

Comment:
could we do something like https://stackoverflow.com/questions/60976342/using-gorm-to-retrieve-tables-names-from-postgresql to make this programmatic?

Reply:
actually this won't affect new tables since those will be automigrated using newer gorm, so a static list is fine.